ibm_mq/datadog_checks/ibm_mq/connection.py

```diff
@@ -58,7 +58,7 @@ def get_ssl_connection(config):
     cd.SSLCipherSpec = config.ssl_cipher_spec
 
     sco = pymqi.SCO()
-    sco.KeyRepository = config.key_repo_location
+    sco.KeyRepository = config.ssl_key_repository_location
```

Comment:
I think this should be https://github.com/DataDog/integrations-core/blob/7169c693140c15d3d7895ca62e8635553cff566d/ibm_mq/datadog_checks/ibm_mq/config.py#L51 `key_repository_location`, no?

Reply:
thanks!